pkg/kubelet/kubelet.go

```diff
@@ -1384,6 +1384,11 @@ func (kl *Kubelet) Run(updates <-chan kubetypes.PodUpdate) {
 		glog.Fatal(err)
 	}
 
+	// Start mounter if it implements mount.MounterWithMountPointsCache.
+	if mounterCache, ok := kl.mounter.(mount.MounterWithMountPointsCache); ok {
```

Comment:
Can you put this behind some configuration flag so we have the option to turn it off if we find an issue with this in production?

Reply:
ok, I'll add it.

Reply:
@msau42 Added in https://github.com/kubernetes/kubernetes/pull/61776/commits/7a245922ec46020261de1b8aac4c795de5cc426d.